assets/src/scss/pages/_listing-page.scss

```diff
@@ -196,6 +196,11 @@
   // Only applied to "grid" layout
   &.wp-block-query--grid {
     .wp-block-post-template {
+      display: flex;
+      flex-direction: row;
+      flex-wrap: wrap;
+      gap: 1.25em;
```

Comment:
Shouldn't we use spacing values here?

Reply:
I used the values from WordPress Query Loop block